sim/src/main/scala/spinal/sim/VerilatorBackend.scala

```diff
@@ -548,10 +615,27 @@ JNIEXPORT void API JNICALL ${jniPrefix}disableWave_1${uniqueId}
     }
   }
 
+  def copyOutputToCache(): Unit = {
+    if (!cacheEnabled) {
```

Comment:
We may also skip the cache update if things were already cached ?

Reply:
moved to the `compileVerilator` method so the cache will only be updated when verilator was run